pkg/pillar/cmd/downloader/syncop.go

```diff
@@ -45,27 +53,18 @@ func handleSyncOp(ctx *downloaderContext, key string,
 	// by default the metricsURL _is_ the DownloadURL, but can override in switch
 	metricsUrl := dsCtx.DownloadURL
 
-	locDirname := types.DownloadDirname + "/" + status.ObjType
-	locFilename = locDirname + "/pending"
-	// XXX common routines to determine pathnames?
-	locFilename = locFilename + "/" + config.ImageID.String()
-
 	// update status to DOWNLOAD STARTED
-	status.FileLocation = locFilename
+	status.FileLocation = locDirname
```

Comment:
We should be able to remove FileLocation from DownloaderStatus. Anything in volumemgr which looks at this should use DownloaderConfig's Target

Reply:
Ha! That was just what you wrote above.

OK, I will remove it.